koku/masu/external/downloader/azure/azure_service.py

```diff
@@ -80,9 +81,9 @@ def _get_latest_blob_for_path(
         self,
         report_path: str,
         container_name: str,
-        extension: str,
+        extensions: list[str] = [AzureBlobExtension.gzip.value, AzureBlobExtension.csv.value],
```

Comment:
We should only pass in the relevant compression type for the report being processed.